board/common/post-build.sh

```diff
@@ -1,5 +1,6 @@
 #!/bin/sh
 # shellcheck disable=SC1090,SC1091
+common=$(dirname "$(readlink -f "$0")")
```

Comment:
```suggestion
```
Nit

Reply:
Thx, will fix that in my next cleanup round.